camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -132,6 +160,23 @@ def __init__(self, boards, service_check = True, synchronizer = message_filters.
         self._pattern = pattern
         self._camera_name = camera_name
         self._max_chessboard_speed = max_chessboard_speed
+        self._no_gui = no_gui
+        self._no_mono_if_stereo = no_mono_if_stereo
+
+        if self._no_mono_if_stereo:
```

Comment:
I dont fully understand the relation between subscribing to the info topics if we dont to monocular calibration

Reply:
You need to load intrinsic parameters somehow... if you do not calculate them, then you need to read them.

Reply:
Search for stereoCalibrate() in [docs](https://docs.opencv.org/4.5.2/d9/d0c/group__calib3d.html#ga91018d80e2a93ade37539f01e6f07de5).

Reply:
thanks, I missed that!